.travis.yml

```diff
@@ -13,9 +12,7 @@ install:
   - pip install -U pip setuptools
   - pip install tox-travis
   - pip install codecov
-  - if [[ $TRAVIS_PYTHON_VERSION != 3.4 ]]; then pip install pre-commit; fi
 script:
   - tox
-  - if [[ $TRAVIS_PYTHON_VERSION != 3.4 ]]; then pre-commit run --all-files; fi
```

Comment:
The whole line was removed but command `pre-commit run --all-files` needs to be there.

Reply:
same